tutorials/clinical/10_ieeg_localize.py

```diff
@@ -203,44 +202,28 @@ def plot_overlay(image, compare, title, thresh=None):
 #
 # .. note:: MNE represents data in the "head" space internally
 
-# load electrophysiology data
+# load electrophysiology data with channel locations
 raw = mne.io.read_raw(op.join(misc_path, 'seeg', 'sample_seeg_ieeg.fif'))
 
-# load electrode positions from file
-elec_df = pd.read_csv(op.join(misc_path, 'seeg', 'sample_seeg_electrodes.tsv'),
-                      sep='\t', header=0, index_col=None)
-ch_names = elec_df['name'].tolist()
-ch_coords = elec_df[['R', 'A', 'S']].to_numpy(dtype=float)
+# MNE stores digitization montages in a coordinate frame called "head"
+# thus even though the channel positions were found in the reference frame of
+# Freesurfer surfaces when they were assigned to the ``raw`` object, they were
+# transformed to "head" so we need to have a transform to put them back
+subj_trans = mne.read_trans(op.join(misc_path, 'seeg',
+                                    'sample_seeg_trans.fif'))
```

Comment:
sorry I saw this late. Is it kosher to have sEEG data as fif files and thus be transformed to "head" coordinates? Don't you normally get the coordinates in MRI coordinates as a separate text file or mat file?

Reply:
Yeah I agree it's not ideal but I think to be backward compatible with MEG/EEG, FIF requires it to be in the "head" frame. I've thought it a bit like how Freesurfer uses surface RAS as its internal representation frame, MNE uses "head". They are all arbitrary reference frames and at least there is a standard and you know how it's stored. Ideally the coordinate frame could be specified in the FIF but that seems like pretty large change.